package.json

```diff
@@ -36,6 +36,13 @@
   },
   "homepage": "https://github.com/1-Platform/one-platform#readme",
   "scripts": {
+    "clean:reports": "rm -R -f e2e/reports && mkdir e2e/reports",
+    "pretest": "npm run clean:reports ",
+    "test:e2e": "cypress run",
+    "combine-reports": "mochawesome-merge --reportDir e2e/reports > e2e/reports/mochareports/report.json",
+    "generate-report": "marge e2e/reports/mochareports/report.json -f report -o e2e/reports/mochareports",
+    "e2e:reports": "npm run combine-reports && npm run generate-report",
+    "test:e2e:reports": "npm run test:e2e || npm run e2e:reports",
```

Comment:
Did you meant `npm run test:e2e && npm run e2e:reports` here?, do you want just one to be run at a time? If the first one fails, second wont be executed.

Reply:
If you want to execute one by one you can do npm run test:e2e and then follow the commands- combine-reports & generate-report. All the commands are defined separately too.

Reply:
If you want to make sure that the second command is not executed if the first one fails then you need to use `&&` operator

Reply:
@gautamkrishnar For tests, it makes more sense to use `||` though.

As we want the reports to be generated even when the tests fail. `&&` usually doesn't execute the second command if the first one fails.

Reply:
If that is the case then its ok. 👍🏻